### DISCUSSION TITLE
Web GUI not accessible

### DISCUSSION THREAD

**Discussion by DanielSpalenka** [haugene/docker-transmission-openvpn#2369] [Q&A]
Hi,

I´ve successfully installed the container via portainer and it runs as it should (as per portainer), however I can´t access the web GUI neither via internally localhost (or 10.0.0.42:9091) nor externally via xx.xxx.xx.xxx:9091. Solutions shown via google search are not working.

Is there somebody who could check the configuration and let me know how t

config files are attached

thanks

Daniel

[config.txt](https://github.com/haugene/docker-transmission-openvpn/files/9795207/config.txt)
[_transmission-openvpn_logs (1).txt](https://github.com/haugene/docker-transmission-openvpn/files/9795208/_transmission-openvpn_logs.1.txt)

**Comment by pkishino** [comment]
externally you mean, via the vpn ip the container is assigned?

"OPENVPN_CONFIG=node-cz-03.protonvpn.net.udp.ovpn",
is wrong..if you check the log it states "Do not specify .ovpn as part of the config name"
remove .ovpn

**Comment by DanielSpalenka** [comment]
Thanks Patrick,

 

that was it. Now it seems the server is starting “healthy” and I can access the web interface.

 

However, now, when including a torrent the transmission doesn´t start since it doesn’t include the torrent into the queue? Any ideas as to why?

 

Daniel 

 

Von: Patrick Kishino ***@***.***> 
Gesendet: Montag, 17. Oktober 2022 04:34
An: haugene/docker-transmission-openvpn ***@***.***>
Cc: DanielSpalenka ***@***.***>; Author ***@***.***>
Betreff: Re: [haugene/docker-transmission-openvpn] Web GUI not accessible (Discussion #2369)

 

externally you mean, via the vpn ip the container is assigned?

"OPENVPN_CONFIG=node-cz-03.protonvpn.net.udp.ovpn",
is wrong..if you check the log it states "Do not specify .ovpn as part of the config name"
remove .ovpn

—
Reply to this email directly, view it on GitHub <https://github.com/haugene/docker-transmission-openvpn/discussions/2369#discussioncomment-3892299> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/A3NOD433SCIQHA4TAJIUBWLWDS3K3ANCNFSM6AAAAAARGNKFWY> .
You are receiving this because you authored the thread.  <https://github.com/notifications/beacon/A3NOD47OX6AXMZMOYO4RPWDWDS3K3A5CNFSM6AAAAAARGNKFW2WGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAHNSEW.gif> Message ID: ***@***.*** ***@***.***> >


No bundled config script found for PROTONVPN. Defaulting to external config
Downloading configs from https://github.com/haugene/vpn-configs-contrib/archive/main.zip into /tmp/tmp.4J3Cs8hlG8
Extracting configs to /tmp/tmp.V282eXLk4T
Found configs for PROTONVPN in /tmp/tmp.V282eXLk4T/vpn-configs-contrib-main/openvpn/protonvpn, will replace current content in /etc/openvpn/protonvpn
Cleanup: deleting /tmp/tmp.4J3Cs8hlG8 and /tmp/tmp.V282eXLk4T
Starting OpenVPN using config node-cz-03.protonvpn.net.udp.ovpn
Modifying /etc/openvpn/protonvpn/node-cz-03.protonvpn.net.udp.ovpn for best behaviour in this container
Modification: Point auth-user-pass option to the username/password file
Modification: Change ca certificate path
Modification: Change ping options
Modification: Update/set resolv-retry to 15 seconds
Modification: Change tls-crypt keyfile path
Modification: Set output verbosity to 3
Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
Setting OpenVPN credentials...
adding route to local network 10.0.0.0/24 via 172.17.0.1 dev eth0
Mon Oct 17 04:26:54 2022 OpenVPN 2.4.7 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Mar 22 2022
Mon Oct 17 04:26:54 2022 library versions: OpenSSL 1.1.1f  31 Mar 2020, LZO 2.10
Mon Oct 17 04:26:54 2022 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Mon Oct 17 04:26:54 2022 Outgoing Control Channel Authentication: Using 512 bit message hash 'SHA512' for HMAC authentication
Mon Oct 17 04:26:54 2022 Incoming Control Channel Authentication: Using 512 bit message hash 'SHA512' for HMAC authentication
Mon Oct 17 04:26:54 2022 TCP/UDP: Preserving recently used remote address: [AF_INET]217.138.220.194:51820
Mon Oct 17 04:26:54 2022 Socket Buffers: R=[212992->212992] S=[212992->212992]
Mon Oct 17 04:26:54 2022 UDP link local: (not bound)
Mon Oct 17 04:26:54 2022 UDP link remote: [AF_INET]217.138.220.194:51820
Mon Oct 17 04:26:54 2022 TLS: Initial packet from [AF_INET]217.138.220.194:51820, sid=1888c4f6 3139c25a
Mon Oct 17 04:26:54 2022 WARNING: this configuration may cache passwords in memory -- use the auth-nocache option to prevent this
Mon Oct 17 04:26:54 2022 VERIFY OK: depth=2, C=CH, O=ProtonVPN AG, CN=ProtonVPN Root CA
Mon Oct 17 04:26:54 2022 VERIFY OK: depth=1, C=CH, O=ProtonVPN AG, CN=ProtonVPN Intermediate CA 1
Mon Oct 17 04:26:54 2022 VERIFY KU OK
Mon Oct 17 04:26:54 2022 Validating certificate extended key usage
Mon Oct 17 04:26:54 2022 ++ Certificate has EKU (str) TLS Web Client Authentication, expects TLS Web Server Authentication
Mon Oct 17 04:26:54 2022 ++ Certificate has EKU (oid) 1.3.6.1.5.5.7.3.2, expects TLS Web Server Authentication
Mon Oct 17 04:26:54 2022 ++ Certificate has EKU (str) 1.3.6.1.5.5.8.2.2, expects TLS Web Server Authentication
Mon Oct 17 04:26:54 2022 ++ Certificate has EKU (oid) 1.3.6.1.5.5.8.2.2, expects TLS Web Server Authentication
Mon Oct 17 04:26:54 2022 ++ Certificate has EKU (str) TLS Web Server Authentication, expects TLS Web Server Authentication
Mon Oct 17 04:26:54 2022 VERIFY EKU OK
Mon Oct 17 04:26:54 2022 VERIFY OK: depth=0, CN=node-cz-03.protonvpn.net
Mon Oct 17 04:26:54 2022 WARNING: 'link-mtu' is used inconsistently, local='link-mtu 1633', remote='link-mtu 1634'
Mon Oct 17 04:26:54 2022 WARNING: 'comp-lzo' is present in remote config but missing in local config, remote='comp-lzo'
Mon Oct 17 04:26:54 2022 Control Channel: TLSv1.3, cipher TLSv1.3 TLS_AES_256_GCM_SHA384, 4096 bit RSA
Mon Oct 17 04:26:54 2022 [node-cz-03.protonvpn.net] Peer Connection Initiated with [AF_INET]217.138.220.194:51820
Mon Oct 17 04:26:55 2022 SENT CONTROL [node-cz-03.protonvpn.net]: 'PUSH_REQUEST' (status=1)
Mon Oct 17 04:26:55 2022 PUSH: Received control message: 'PUSH_REPLY,dhcp-option DNS 10.19.0.1,sndbuf 524288,rcvbuf 524288,redirect-gateway def1,explicit-exit-notify,comp-lzo no,route-gateway 10.19.0.1,topology subnet,ping 10,ping-restart 60,socket-flags TCP_NODELAY,ifconfig 10.19.0.5 255.255.0.0,peer-id 196611,cipher AES-256-GCM'
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: timers and/or timeouts modified
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: explicit notify parm(s) modified
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: compression parms modified
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: --sndbuf/--rcvbuf options modified
Mon Oct 17 04:26:55 2022 Socket Buffers: R=[212992->425984] S=[212992->425984]
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: --socket-flags option modified
Mon Oct 17 04:26:55 2022 NOTE: setsockopt TCP_NODELAY=1 failed
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: --ifconfig/up options modified
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: route options modified
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: route-related options modified
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: --ip-win32 and/or --dhcp-option options modified
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: peer-id set
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: adjusting link_mtu to 1656
Mon Oct 17 04:26:55 2022 OPTIONS IMPORT: data channel crypto options modified
Mon Oct 17 04:26:55 2022 Data Channel: using negotiated cipher 'AES-256-GCM'
Mon Oct 17 04:26:55 2022 Outgoing Data Channel: Cipher 'AES-256-GCM' initialized with 256 bit key
Mon Oct 17 04:26:55 2022 Incoming Data Channel: Cipher 'AES-256-GCM' initialized with 256 bit key
Mon Oct 17 04:26:55 2022 ROUTE_GATEWAY 172.17.0.1/255.255.0.0 IFACE=eth0 HWADDR=02:42:ac:11:00:04
Mon Oct 17 04:26:55 2022 TUN/TAP device tun0 opened
Mon Oct 17 04:26:55 2022 TUN/TAP TX queue length set to 100
Mon Oct 17 04:26:55 2022 /sbin/ip link set dev tun0 up mtu 1500
Mon Oct 17 04:26:55 2022 /sbin/ip addr add dev tun0 10.19.0.5/16 broadcast 10.19.255.255
Mon Oct 17 04:26:55 2022 /etc/openvpn/tunnelUp.sh tun0 1500 1584 10.19.0.5 255.255.0.0 init
Up script executed with tun0 1500 1584 10.19.0.5 255.255.0.0 init
Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.19.0.5
Updating Transmission settings.json with values from env variables
Using existing settings.json for Transmission /data/transmission-home/settings.json
Overriding bind-address-ipv4 because TRANSMISSION_BIND_ADDRESS_IPV4 is set to 10.19.0.5
Overriding download-dir because TRANSMISSION_DOWNLOAD_DIR is set to /media/torrents/completed
Overriding incomplete-dir because TRANSMISSION_INCOMPLETE_DIR is set to /media/torrents/incomplete
Overriding rpc-port because TRANSMISSION_RPC_PORT is set to 9091
Overriding watch-dir because TRANSMISSION_WATCH_DIR is set to /media/torrents/watch
sed'ing True to true
Enforcing ownership on transmission config directories
Applying permissions to transmission config directories
Setting owner for transmission paths to 1000:1000
Setting permissions for download and incomplete directories

2
Directories: 775
Files: 664
Setting permission for watch directory (775) and its files (664)

-------------------------------------
Transmission will run as
-------------------------------------
User name:   abc
User uid:    1000
User gid:    1000
-------------------------------------

STARTING TRANSMISSION
Transmission startup script complete.
Mon Oct 17 04:26:56 2022 /sbin/ip route add 217.138.220.194/32 via 172.17.0.1
Mon Oct 17 04:26:56 2022 /sbin/ip route add 0.0.0.0/1 via 10.19.0.1
Mon Oct 17 04:26:56 2022 /sbin/ip route add 128.0.0.0/1 via 10.19.0.1
Mon Oct 17 04:26:56 2022 Initialization Sequence Completed

{
    "AppArmorProfile": "docker-default",
    "Args": [
        "/etc/openvpn/start.sh"
    ],
    "Config": {
        "AttachStderr": false,
        "AttachStdin": false,
        "AttachStdout": false,
        "Cmd": [
            "dumb-init",
            "/etc/openvpn/start.sh"
        ],
        "Domainname": "",
        "Entrypoint": null,
        "Env": [
            "OPENVPN_PASSWORD=Aslan123",
            "LOCAL_NETWORK=10.0.0.0/24",
            "OPENVPN_PROVIDER=PROTONVPN",
            "OPENVPN_CONFIG=node-cz-03.protonvpn.net.udp",
            "OPENVPN_USERNAME=Daniel.Spalenka",
            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
            "GLOBAL_APPLY_PERMISSIONS=true",
            "TRANSMISSION_HOME=/data/transmission-home",
            "TRANSMISSION_RPC_PORT=9091",
            "TRANSMISSION_DOWNLOAD_DIR=/media/torrents/completed",
            "TRANSMISSION_INCOMPLETE_DIR=/media/torrents/incomplete",
            "TRANSMISSION_WATCH_DIR=/media/torrents/watch",
            "CREATE_TUN_DEVICE=true",
            "ENABLE_UFW=false",
            "UFW_ALLOW_GW_NET=false",
            "UFW_EXTRA_PORTS=",
            "UFW_DISABLE_IPTABLES_REJECT=false",
            "PUID=1000",
            "PGID=1000",
            "PEER_DNS=true",
            "PEER_DNS_PIN_ROUTES=true",
            "DROP_DEFAULT_ROUTE=",
            "WEBPROXY_ENABLED=false",
            "WEBPROXY_PORT=8118",
            "WEBPROXY_USERNAME=",
            "WEBPROXY_PASSWORD=",
            "LOG_TO_STDOUT=false",
            "HEALTH_CHECK_HOST=google.com",
            "SELFHEAL=false",
            "REVISION=4fd75ea3e84d3a1a9d1f5d4a0a8e5d0eebbb8f89",
            "TRANSMISSION_WEB_HOME=/combustion-release/"
        ],
        "ExposedPorts": {
            "8118/tcp": {},
            "9091/tcp": {}
        },
        "Healthcheck": {
            "Interval": 60000000000,
            "Test": [
                "CMD-SHELL",
                "/etc/scripts/healthcheck.sh"
            ]
        },
        "Hostname": "2e48af0e3e9c",
        "Image": "haugene/transmission-openvpn:latest",
        "Labels": {
            "autoheal": "true",
            "org.opencontainers.image.revision": "4fd75ea3e84d3a1a9d1f5d4a0a8e5d0eebbb8f89",
            "org.opencontainers.image.source": "https://github.com/haugene/docker-transmission-openvpn"
        },
        "OnBuild": null,
        "OpenStdin": false,
        "StdinOnce": false,
        "Tty": false,
        "User": "",
        "Volumes": {
            "/config": {},
            "/data": {}
        },
        "WorkingDir": ""
    },
    "Created": "2022-10-17T04:26:49.066600264Z",
    "Driver": "overlay2",
    "ExecIDs": null,
    "GraphDriver": {
        "Data": {
            "LowerDir": "/var/lib/docker/overlay2/58744e53e49c7ea4bb757c5cac90ec4e9e54874ef59d7a1ca96fb46d088090f5-init/diff:/var/lib/docker/overlay2/1bd050e8d2228d87a210ad47fadaaed78953437e4c6c3c51fa9df38bc122ea4e/diff:/var/lib/docker/overlay2/b237c918f3429dc81915bd4198a6d6f9bd260172e5b9319654b48752e0a80bb7/diff:/var/lib/docker/overlay2/e1451f85559601ea97ba9515f1b783b871a4d610b4da97553b8f3b93331efc8e/diff:/var/lib/docker/overlay2/7c5d6506266f391bcf5e49d03f90e6c5e9723bd5ab073618a344d3f5096e4d40/diff:/var/lib/docker/overlay2/bcf9d5e10ff5aa25d8b8186ddf2ba502c5ba4adbeae76a4c41aa278e62acdcfd/diff:/var/lib/docker/overlay2/ef848f37d2e468b017dedcc4a01ad962b3cb3bf28a9e9d04575a82b9d8145064/diff:/var/lib/docker/overlay2/cde7e20c6f69cbadf3ff757a93f3c72b75f3401c1989dc9002f57768f3c15519/diff",
            "MergedDir": "/var/lib/docker/overlay2/58744e53e49c7ea4bb757c5cac90ec4e9e54874ef59d7a1ca96fb46d088090f5/merged",
            "UpperDir": "/var/lib/docker/overlay2/58744e53e49c7ea4bb757c5cac90ec4e9e54874ef59d7a1ca96fb46d088090f5/diff",
            "WorkDir": "/var/lib/docker/overlay2/58744e53e49c7ea4bb757c5cac90ec4e9e54874ef59d7a1ca96fb46d088090f5/work"
        },
        "Name": "overlay2"
    },
    "HostConfig": {
        "AutoRemove": false,
        "Binds": [
            "/media/torrents:/data",
            "89c45da6fe6816814ad7c3a3465e42841b646569c33d70f8f5fb6fbdb34542ce:/config"
        ],
        "BlkioDeviceReadBps": null,
        "BlkioDeviceReadIOps": null,
        "BlkioDeviceWriteBps": null,
        "BlkioDeviceWriteIOps": null,
        "BlkioWeight": 0,
        "BlkioWeightDevice": [],
        "CapAdd": [
            "AUDIT_WRITE",
            "CHOWN",
            "DAC_OVERRIDE",
            "FOWNER",
            "FSETID",
            "KILL",
            "MKNOD",
            "NET_ADMIN",
            "NET_BIND_SERVICE",
            "NET_RAW",
            "SETFCAP",
            "SETGID",
            "SETPCAP",
            "SETUID",
            "SYS_CHROOT"
        ],
        "CapDrop": [
            "AUDIT_CONTROL",
            "BLOCK_SUSPEND",
            "DAC_READ_SEARCH",
            "IPC_LOCK",
            "IPC_OWNER",
            "LEASE",
            "LINUX_IMMUTABLE",
            "MAC_ADMIN",
            "MAC_OVERRIDE",
            "NET_BROADCAST",
            "SYSLOG",
            "SYS_ADMIN",
            "SYS_BOOT",
            "SYS_MODULE",
            "SYS_NICE",
            "SYS_PACCT",
            "SYS_PTRACE",
            "SYS_RAWIO",
            "SYS_RESOURCE",
            "SYS_TIME",
            "SYS_TTY_CONFIG",
            "WAKE_ALARM"
        ],
        "Cgroup": "",
        "CgroupParent": "",
        "CgroupnsMode": "private",
        "ConsoleSize": [
            0,
            0
        ],
        "ContainerIDFile": "",
        "CpuCount": 0,
        "CpuPercent": 0,
        "CpuPeriod": 0,
        "CpuQuota": 0,
        "CpuRealtimePeriod": 0,
        "CpuRealtimeRuntime": 0,
        "CpuShares": 0,
        "CpusetCpus": "",
        "CpusetMems": "",
        "DeviceCgroupRules": null,
        "DeviceRequests": null,
        "Devices": [],
        "Dns": [],
        "DnsOptions": [],
        "DnsSearch": [],
        "ExtraHosts": [],
        "GroupAdd": null,
        "IOMaximumBandwidth": 0,
        "IOMaximumIOps": 0,
        "IpcMode": "private",
        "Isolation": "",
        "KernelMemory": 0,
        "KernelMemoryTCP": 0,
        "Links": null,
        "LogConfig": {
            "Config": {
                "max-size": "10m"
            },
            "Type": "json-file"
        },
        "MaskedPaths": [
            "/proc/asound",
            "/proc/acpi",
            "/proc/kcore",
            "/proc/keys",
            "/proc/latency_stats",
            "/proc/timer_list",
            "/proc/timer_stats",
            "/proc/sched_debug",
            "/proc/scsi",
            "/sys/firmware"
        ],
        "Memory": 0,
        "MemoryReservation": 0,
        "MemorySwap": 0,
        "MemorySwappiness": null,
        "NanoCpus": 0,
        "NetworkMode": "bridge",
        "OomKillDisable": null,
        "OomScoreAdj": 0,
        "PidMode": "",
        "PidsLimit": null,
        "PortBindings": {
            "9091/tcp": [
                {
                    "HostIp": "",
                    "HostPort": "9091"
                }
            ]
        },
        "Privileged": false,
        "PublishAllPorts": false,
        "ReadonlyPaths": [
            "/proc/bus",
            "/proc/fs",
            "/proc/irq",
            "/proc/sys",
            "/proc/sysrq-trigger"
        ],
        "ReadonlyRootfs": false,
        "RestartPolicy": {
            "MaximumRetryCount": 0,
            "Name": "unless-stopped"
        },
        "Runtime": "runc",
        "SecurityOpt": null,
        "ShmSize": 67108864,
        "UTSMode": "",
        "Ulimits": null,
        "UsernsMode": "",
        "VolumeDriver": "",
        "VolumesFrom": null
    },
    "HostnamePath": "/var/lib/docker/containers/2af31342753e564428eb327a6d25a1a8e7ba1e329c677f6f7f3f066fc7bc8dce/hostname",
    "HostsPath": "/var/lib/docker/containers/2af31342753e564428eb327a6d25a1a8e7ba1e329c677f6f7f3f066fc7bc8dce/hosts",
    "Id": "2af31342753e564428eb327a6d25a1a8e7ba1e329c677f6f7f3f066fc7bc8dce",
    "Image": "sha256:c9fcfb332f591b16ae433b6d224e4731ff9e7f6ed61c6252c251f915e7d3260e",
    "LogPath": "/var/lib/docker/containers/2af31342753e564428eb327a6d25a1a8e7ba1e329c677f6f7f3f066fc7bc8dce/2af31342753e564428eb327a6d25a1a8e7ba1e329c677f6f7f3f066fc7bc8dce-json.log",
    "MountLabel": "",
    "Mounts": [
        {
            "Destination": "/config",
            "Driver": "local",
            "Mode": "z",
            "Name": "89c45da6fe6816814ad7c3a3465e42841b646569c33d70f8f5fb6fbdb34542ce",
            "Propagation": "",
            "RW": true,
            "Source": "/var/lib/docker/volumes/89c45da6fe6816814ad7c3a3465e42841b646569c33d70f8f5fb6fbdb34542ce/_data",
            "Type": "volume"
        },
        {
            "Destination": "/data",
            "Mode": "",
            "Propagation": "rprivate",
            "RW": true,
            "Source": "/media/torrents",
            "Type": "bind"
        }
    ],
    "Name": "/transmission-openvpn",
    "NetworkSettings": {
        "Bridge": "",
        "EndpointID": "d1a1ce2cad3888af0b8cd5b40cd2b8baa697d3f26c047d796999f01e24bfb031",
        "Gateway": "172.17.0.1",
        "GlobalIPv6Address": "",
        "GlobalIPv6PrefixLen": 0,
        "HairpinMode": false,
        "IPAddress": "172.17.0.4",
        "IPPrefixLen": 16,
        "IPv6Gateway": "",
        "LinkLocalIPv6Address": "",
        "LinkLocalIPv6PrefixLen": 0,
        "MacAddress": "02:42:ac:11:00:04",
        "Networks": {
            "bridge": {
                "Aliases": null,
                "DriverOpts": null,
                "EndpointID": "d1a1ce2cad3888af0b8cd5b40cd2b8baa697d3f26c047d796999f01e24bfb031",
                "Gateway": "172.17.0.1",
                "GlobalIPv6Address": "",
                "GlobalIPv6PrefixLen": 0,
                "IPAMConfig": {},
                "IPAddress": "172.17.0.4",
                "IPPrefixLen": 16,
                "IPv6Gateway": "",
                "Links": null,
                "MacAddress": "02:42:ac:11:00:04",
                "NetworkID": "dc411232cfdcc19c0e24ac2152fcc6e39b830940ed874d1624910b3904214794"
            }
        },
        "Ports": {
            "8118/tcp": null,
            "9091/tcp": [
                {
                    "HostIp": "0.0.0.0",
                    "HostPort": "9091"
                },
                {
                    "HostIp": "::",
                    "HostPort": "9091"
                }
            ]
        },
        "SandboxID": "503e0a78cfff0d6df42848c58188d31566c975abc9466713f0b2b4383b44b8b6",
        "SandboxKey": "/var/run/docker/netns/503e0a78cfff",
        "SecondaryIPAddresses": null,
        "SecondaryIPv6Addresses": null
    },
    "Path": "dumb-init",
    "Platform": "linux",
    "Portainer": {
        "ResourceControl": {
            "Id": 28,
            "ResourceId": "2af31342753e564428eb327a6d25a1a8e7ba1e329c677f6f7f3f066fc7bc8dce",
            "SubResourceIds": [],
            "Type": 1,
            "UserAccesses": [],
            "TeamAccesses": [],
            "Public": false,
            "AdministratorsOnly": true,
            "System": false
        }
    },
    "ProcessLabel": "",
    "ResolvConfPath": "/var/lib/docker/containers/2af31342753e564428eb327a6d25a1a8e7ba1e329c677f6f7f3f066fc7bc8dce/resolv.conf",
    "RestartCount": 0,
    "State": {
        "Dead": false,
        "Error": "",
        "ExitCode": 0,
        "FinishedAt": "0001-01-01T00:00:00Z",
        "Health": {
            "FailingStreak": 0,
            "Log": [
                {
                    "End": "2022-10-17T06:28:53.370929841+02:00",
                    "ExitCode": 0,
                    "Output": "PING google.com (142.251.37.110) 56(84) bytes of data.\n64 bytes from prg03s13-in-f14.1e100.net (142.251.37.110): icmp_seq=1 ttl=119 time=19.0 ms\n64 bytes from prg03s13-in-f14.1e100.net (142.251.37.110): icmp_seq=2 ttl=119 time=19.6 ms\n\n--- google.com ping statistics ---\n2 packets transmitted, 2 received, 0% packet loss, time 1001ms\nrtt min/avg/max/mdev = 19.002/19.320/19.638/0.318 ms\nNetwork is up\nOpenvpn and transmission-daemon processes are running\n",
                    "Start": "2022-10-17T06:28:52.198617894+02:00"
                },
                {
                    "End": "2022-10-17T06:29:54.593852325+02:00",
                    "ExitCode": 0,
                    "Output": "PING google.com (142.251.37.110) 56(84) bytes of data.\n64 bytes from prg03s13-in-f14.1e100.net (142.251.37.110): icmp_seq=1 ttl=119 time=19.4 ms\n64 bytes from prg03s13-in-f14.1e100.net (142.251.37.110): icmp_seq=2 ttl=119 time=19.1 ms\n\n--- google.com ping statistics ---\n2 packets transmitted, 2 received, 0% packet loss, time 1002ms\nrtt min/avg/max/mdev = 19.142/19.268/19.394/0.126 ms\nNetwork is up\nOpenvpn and transmission-daemon processes are running\n",
                    "Start": "2022-10-17T06:29:53.424109823+02:00"
                },
                {
                    "End": "2022-10-17T06:30:55.817119139+02:00",
                    "ExitCode": 0,
                    "Output": "PING google.com (142.251.37.110) 56(84) bytes of data.\n64 bytes from prg03s13-in-f14.1e100.net (142.251.37.110): icmp_seq=1 ttl=119 time=19.9 ms\n64 bytes from prg03s13-in-f14.1e100.net (142.251.37.110): icmp_seq=2 ttl=119 time=19.8 ms\n\n--- google.com ping statistics ---\n2 packets transmitted, 2 received, 0% packet loss, time 1002ms\nrtt min/avg/max/mdev = 19.768/19.825/19.883/0.057 ms\nNetwork is up\nOpenvpn and transmission-daemon processes are running\n",
                    "Start": "2022-10-17T06:30:54.649035834+02:00"
                },
                {
                    "End": "2022-10-17T06:31:57.059777739+02:00",
                    "ExitCode": 0,
                    "Output": "PING google.com (142.251.37.110) 56(84) bytes of data.\n64 bytes from prg03s13-in-f14.1e100.net (142.251.37.110): icmp_seq=1 ttl=119 time=20.0 ms\n64 bytes from prg03s13-in-f14.1e100.net (142.251.37.110): icmp_seq=2 ttl=119 time=19.5 ms\n\n--- google.com ping statistics ---\n2 packets transmitted, 2 received, 0% packet loss, time 1002ms\nrtt min/avg/max/mdev = 19.484/19.752/20.021/0.268 ms\nNetwork is up\nOpenvpn and transmission-daemon processes are running\n",
                    "Start": "2022-10-17T06:31:55.877169513+02:00"
                },
                {
                    "End": "2022-10-17T06:32:58.301512526+02:00",
                    "ExitCode": 0,
                    "Output": "PING google.com (142.251.37.110) 56(84) bytes of data.\n64 bytes from prg03s13-in-f14.1e100.net (142.251.37.110): icmp_seq=1 ttl=119 time=19.7 ms\n64 bytes from prg03s13-in-f14.1e100.net (142.251.37.110): icmp_seq=2 ttl=119 time=19.9 ms\n\n--- google.com ping statistics ---\n2 packets transmitted, 2 received, 0% packet loss, time 1001ms\nrtt min/avg/max/mdev = 19.665/19.784/19.904/0.119 ms\nNetwork is up\nOpenvpn and transmission-daemon processes are running\n",
                    "Start": "2022-10-17T06:32:57.12754479+02:00"
                }
            ],
            "Status": "healthy"
        },
        "OOMKilled": false,
        "Paused": false,
        "Pid": 114167,
        "Restarting": false,
        "Running": true,
        "StartedAt": "2022-10-17T04:26:50.943664616Z",
        "Status": "running"
    }
}

**Reply by pkishino** [reply]
as per your log the container is up properly.. you might need to check the transmission logs or so for errors if you cannot add a torrent, possible file corrupted

**Comment by DanielSpalenka** [comment]
Thank you for following up.

 

Unfortunately, the issue persist, i.e. the GUI doesn´t show. Maybe I have to connect from WAN via the VPN IP address but I don´t now what IP address that would be?

 

Cheers

 

Daniel

 

Von: Patrick Kishino ***@***.***> 
Gesendet: Dienstag, 18. Oktober 2022 04:26
An: haugene/docker-transmission-openvpn ***@***.***>
Cc: DanielSpalenka ***@***.***>; Author ***@***.***>
Betreff: Re: [haugene/docker-transmission-openvpn] Web GUI not accessible (Discussion #2369)

 

as per your log the container is up properly.. you might need to check the transmission logs or so for errors if you cannot add a torrent, possible file corrupted

**Reply by pkishino** [reply]
No, you definitely don’t want to allow access via wan.. if you cannot connect to the GUI then most likely your LOCAL_NETWORK isn’t set correctly. Check the documentation
